package.json

```diff
@@ -16,8 +16,12 @@
   },
   "dependencies": {
     "@netdata/netdata-ui": "^0.7.30",
+    "ramda": "^0.27.0",
+    "react": "^16.13.1",
+    "react-dom": "^16.13.1",
```

Comment:
what about the stuff in peerDependencies, are we gonna have duplicates here? Will it work? I'm afraid that we'll have errors from React about second instance